admission-controller/synchronizer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:18.16.0-bullseye-slim
+
+WORKDIR /workdir
+COPY package*.json ./
```

Comment:
Nope, this is an anti-pattern which will result in both a large, bloated image as well as possibly insecure due to having all of these dependencies installed.
Use multi-stage builds like in [the Validator builds](https://github.com/kubeshop/monokle-saas/blob/main/apiDomain/validationService/Dockerfile_monokle).

Reply:
Fixed in https://github.com/kubeshop/monokle-admission-controller/pull/12/commits/fa6403fcb7ca136d1c4fa3aa3cce5259ea837379.